app/client/views/resumes/resumes.html

```diff
@@ -0,0 +1,207 @@
+<div id="dashboard" class="page">
+  <div class="divided title">
+    Resumes
+  </div>
+
+  <div class="ui segment">
+
+    <div class="ui stackable grid">
+      <div class="three wide column">
+        <div class="ui header">
+          Search
+        </div>
+        <div class="ui form">
+          <div class="field">
+            <div class="ui icon fluid input">
+              <input type="text" placeholder="Search..." ng-model="queryText" ng-model-options="{debounce: 300}">
+              <i class="search icon"></i>
+            </div>
+          </div>
+
+        </div>
+
+        <div class="ui divider"></div>
+
+        <button ng-repeat="page in pages" ng-class="page == currentPage ? 'active pink' : ''" ng-click="goToPage(page)"
+          class="ui icon basic page button">
+          {{page}}
+        </button>
+
+      </div>
+      <div id="table-container" class="thirteen wide column">
+        <div class="ui header">
+          Users
+        </div>
+        <div class="ui long user modal">
+          <i class="close icon"></i>
+          <div class="header">
+            {{selectedUser.profile.name ? selectedUser.profile.name : selectedUser.email}}
+            <span ng-show="selectedUser.admin">
+              (Admin)
+            </span>
+          </div>
+          <div class="content">
+
+            <div>
+              <p>
+                <strong> Resume: </strong>
+              </p>
+
+              <canvas id="resume-canvas"></canvas>
+            </div>
```

Comment:
nit: looks like this content is the only difference between admin/users/users.html

Ideally we can have a reusable component so we don't re-write this and have to maintain both

Reply:
I updated resumes.html to remove a lot of the admin functionalities that a sponsor shouldn't have, so it's a much more lean version of admin/users/users.html now. What should I do about this file now?